packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -86,9 +87,58 @@ class _DriverBinding extends BindingBase with SchedulerBinding, ServicesBinding,
 /// for tests where exceptions are expected. Defaults to false. Any errors
 /// will still be returned in the `response` field of the result JSON along
 /// with an `isError` boolean.
-void enableFlutterDriverExtension({ DataHandler handler, bool silenceErrors = false }) {
+///
+/// The `finders` parameter are used to add custom finders, as in the following example.
+///
+/// ```dart main
+/// void main() {
+///   enableFlutterDriverExtension(finders: <FinderExtension>[ SomeFinderExtension() ]);
+///
+///   app.main();
+/// }
+/// ```
+///
+/// ```dart
+/// class Some extends SerializableFinder {
+///   const Some();
+///
+///   @override
+///   String get finderType => 'Some';
+///
+///   @override
+///   Map<String, String> serialize() {
+///     // Serialize Finder
+///   }
+/// }
+/// ```
+///
+/// ```dart
+/// class SomeFinderExtension extends FinderExtension {
+///
+///  String get finderType => 'Some';
+///
+///  SerializableFinder deserialize(Map<String, String> params, DeserializeFinderFactory finderFactory) {
+///    SomeFinder result = SomeFinder();
+///    // Deserialize Finder
+///    ...
```

Comment:
Please expand this.

I think I'm seeing it now, but I'm worried that future me will forget this again - and if I were reading the example to figure things out I'd have no idea what goes in here.